From: Agathe Porte <gagath@debian.org>
Subject: remove non-linux dependencies

The debcargo tooling seems unable to detect that the macos and windows-specific
dependencies are not needed. Manually remove them from Cargo.toml so that
d/control does not depend on these unlikely to be packaged crates.

Forwarded: not-needed
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,10 +39,6 @@
     "win_sendinput_send_scancodes",
     "zippychord",
 ]
-interception_driver = [
-    "kanata-interception",
-    "kanata-parser/interception_driver",
-]
 passthru_ahk = [
     "simulated_input",
     "simulated_output",
@@ -52,11 +48,6 @@
 simulated_output = ["indoc"]
 tcp_server = ["serde_json"]
 win_llhook_read_scancodes = ["kanata-parser/win_llhook_read_scancodes"]
-win_manifest = [
-    "embed-resource",
-    "indoc",
-    "regex",
-]
 win_sendinput_send_scancodes = ["kanata-parser/win_sendinput_send_scancodes"]
 zippychord = ["kanata-parser/zippychord"]
 
@@ -163,92 +154,6 @@
 [target.'cfg(target_os = "linux")'.dependencies.signal-hook]
 version = "0.3.14"
 
-[target.'cfg(target_os = "macos")'.dependencies.karabiner-driverkit]
-version = "0.1.5"
-
-[target.'cfg(target_os = "macos")'.dependencies.libc]
-version = "0.2"
-
-[target.'cfg(target_os = "macos")'.dependencies.objc]
-version = "0.2.7"
-
-[target.'cfg(target_os = "macos")'.dependencies.open]
-version = "5"
-optional = true
-
-[target.'cfg(target_os = "macos")'.dependencies.os_pipe]
-version = "1.2.1"
-
-[target.'cfg(target_os = "windows")'.dependencies.encode_unicode]
-version = "0.3.6"
-
-[target.'cfg(target_os = "windows")'.dependencies.kanata-interception]
-version = "0.3.0"
-optional = true
-
-[target.'cfg(target_os = "windows")'.dependencies.muldiv]
-version = "1.0.1"
-optional = true
-
-[target.'cfg(target_os = "windows")'.dependencies.native-windows-gui]
-version = "1.0.13"
-default-features = false
-
-[target.'cfg(target_os = "windows")'.dependencies.open]
-version = "5"
-features = ["shellexecute-on-windows"]
-optional = true
-
-[target.'cfg(target_os = "windows")'.dependencies.regex]
-version = "1.10.4"
-optional = true
-
-[target.'cfg(target_os = "windows")'.dependencies.strip-ansi-escapes]
-version = "0.2.0"
-optional = true
-
-[target.'cfg(target_os = "windows")'.dependencies.winapi]
-version = "0.3.9"
-features = [
-    "wincon",
-    "timeapi",
-    "mmsystem",
-]
-
-[target.'cfg(target_os = "windows")'.dependencies.windows-sys]
-version = "0.52.0"
-features = [
-    "Win32_Devices_DeviceAndDriverInstallation",
-    "Win32_Devices_Usb",
-    "Win32_Foundation",
-    "Win32_Graphics_Gdi",
-    "Win32_Security",
-    "Win32_System_Diagnostics_Debug",
-    "Win32_System_Registry",
-    "Win32_System_Threading",
-    "Win32_UI_Controls",
-    "Win32_UI_Shell",
-    "Win32_UI_HiDpi",
-    "Win32_UI_WindowsAndMessaging",
-    "Win32_System_SystemInformation",
-    "Wdk",
-    "Wdk_System",
-    "Wdk_System_SystemServices",
-]
-optional = true
-
-[target.'cfg(target_os = "windows")'.build-dependencies.embed-resource]
-version = "2.4.2"
-optional = true
-
-[target.'cfg(target_os = "windows")'.build-dependencies.indoc]
-version = "2.0.4"
-optional = true
-
-[target.'cfg(target_os = "windows")'.build-dependencies.regex]
-version = "1.10.4"
-optional = true
-
 [profile.release]
 opt-level = "z"
 lto = "fat"
